etc/kayobe/ansible/scripts/drives/nvme.json

```diff
@@ -0,0 +1,24 @@
+{
+  "device_info": {
+    "name": "/dev/nvme0",
```

Comment:
Perhaps rename the subfolder `drives` to `tests`? And then later on (not in this change) we can move the script to a dedicated repo.